README.md

```diff
@@ -92,15 +92,15 @@ For project using Scala 2.11
 <dependency>
     <groupId>za.co.absa</groupId>
     <artifactId>atum_2.11</artifactId>
-    <version>3.3.0</version>
```

Comment:
The current version of `develop` seems to be 3.5.1 Don't you want to use that one?

Reply:
Done